File: C/1133 Resto da Divisão.c
#include <stdio.h>

int main(void) 
{
  int a, b, i;
  scanf("%d %d", &a, &b);
  if (a > b)
  {
    int aux = a;
    a = b;
    b = aux;
  }
  for (i = a+1; i < b; i++)
    {
      if (i % 5 == 2 || i % 5 == 3)
      {
        printf("%d\n", i);
      }
    }
  return 0;
}

File: C#/1133 Resto da Divisão.cs
using System;

class MainClass 
{
  public static void Main (string[] args) 
  {
    int v1 = int.Parse(Console.ReadLine());
    int v2 = int.Parse(Console.ReadLine());
    if (v1 > v2)
    {
      int aux = v1;
      v1 = v2;
      v2 = aux;
    }
    for (int i = v1+1; i < v2; i++)
    {
      if (i % 5 == 2 || i % 5 == 3)
      {
        Console.WriteLine(i);
      }
    }
  }
}
